app/src/main/res/values/strings.xml

```diff
@@ -83,7 +83,7 @@
     <string name="chat_with_ta">Chat with TA</string>
     <string name="comment">Comment</string>
     <string name="doubts">Doubts</string>
-    <string name="my_courses">My Courses</string>
+    <string name="my_courses">MyCour..</string>
     <string name="explore">Explore</string>
```

Comment:
Thus is not what i meant 🙂it will change the ellipse for every Phone

Reply:
Yeah but only difference is coming for My courses rest tabs are working fine I checked on 3 different sized devices. and this thing works for all the three. Although its an hack but it worked.

Reply:
> Thus is not what i meant 🙂it will change the ellipse for every Phone

@aggarwalpulkit596 sir in the latest commit ellipsize added. pls review